README.md

```
@@ -1,97 +1,349 @@
+npm run typecheck
+
+# Lint
+npm run lint
+
+# Format
+npm run format
+```
+
+### Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)